feature/auth/create_password/build.gradle.kts

```diff
@@ -50,6 +50,9 @@ dependencies {
     implementation(libs.ui.tooling.preview)
     implementation(libs.navigation.compose)
     implementation(libs.lifecycle.runtime.compose)
+    implementation(libs.koin.androidx.compose)
+    implementation(project(mapOf("path" to ":data:auth")))
+    implementation(project(mapOf("path" to ":core:network")))
```

Comment:
Use implementation(project(":core:network)) at the end of file, and remove this line